dist/s_void

```diff
@@ -66,20 +66,54 @@ func_ok()
         -e '/int __time_value_validate_parent_stable$/d' \
         -e '/int __tree_walk_skip_count_callback$/d' \
         -e '/int __txn_rollback_to_stable_custom_skip$/d' \
+        -e '/int __ut_ckpt_add_blkmod_entry$/d' \
+        -e '/int __ut_ovfl_discard_verbose$/d' \
+        -e '/int __ut_ovfl_discard_wrapup$/d' \
         -e '/int __win_terminate$/d' \
         -e '/int __wt_block_compact_end$/d' \
         -e '/int __wt_block_compact_start$/d' \
         -e '/int __wt_block_disagg_manager_create$/d' \
```

Comment:
Should this be removed as well? 

https://github.com/wiredtiger/wiredtiger/blame/cd42e06513c9c82aa29e5a77ac176e247b77b3b5/dist/s_void#L72